src/app/components/whitepagepopup/adminpopup/addpopup/addpopup.css

```diff
@@ -0,0 +1,139 @@
+.popupOverlay {
+    position: fixed;
+    top: 0;
```

Comment:
I'm not sure if the popup has been finished yet, if it has, can you let me know how I can see it?

Reply:
I made it so now when you click the add member that one shows and then with the bubble the edit member one shows